pt_BR.ini

```diff
@@ -18,17 +18,17 @@ DownloadFromStore = Download do repositório de homebrew do PPSSPP
 
 [Game]
 Play = Jogar
-Game Settings = Configurações do jogo
+Game Settings = Configurações
```

Comment:
My mistake, didn’t see this






Sent from Windows Mail





From: papel
Sent: ‎Thursday‎, ‎February‎ ‎5‎, ‎2015 ‎12‎:‎37‎ ‎AM
To: hrydgard/ppsspp-lang
Cc: Gabriel Franco





In pt_BR.ini:
> @@ -18,17 +18,17 @@ DownloadFromStore = Download do repositório de homebrew do PPSSPP
>  
>  [Game]
>  Play = Jogar
> -Game Settings = Configurações do jogo
> +Game Settings = Configurações


It should be "Configurações do jogo", because it is specific settings for the current game. It appears after clicking on Create Game Config.

—
Reply to this email directly or view it on GitHub.